Assets/Scripts/Strategy/BaseManagement/BaseManager.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class BaseManager : MonoBehaviour
+{
+    //List<IBuilding> productionBuildings;
+    //List<IBuilding> resourceBuildings;
+
+    //The first element is the tiers option canvas
+    [SerializeField] List<GameObject> buildingTierCanvases;
+    [SerializeField] List<UnityEngine.UI.Button> tierButtonsList;

```

Comment:
Access modifiers should probably be explicitly defined

Reply:
0e267b2